0006-cmx2cdp-protocol/README.md

```diff
@@ -94,19 +112,19 @@ Example:
 }
 ```
 
-_"config-response"_ message structure:
+`ConfigResponse` message structure:
 
-- `correlationId` (string, required) - see [3/Messaging IPC][3/MIPC] for description.
-- `timestamp` (number, required) - timestamp of the message creation time.
-- `timeout` (number, required) - the amount of time from timestamp that message is actual.
-- `endpointMessageId` (string, required) - unique identifier of original endpoint message.
-- `appVersionName` (string, required) - application version to which endpoint configuration is applicable.
-- `endpointId` (string, required) - endpoint unique identifier to which configuration is applicable.
-- `contentType` (string, required) - content type of configuration, e.g.: json, protobuf.
-- `configId` (string, required) - unique identifier of endpoint configuration.
-- `statusCode` (number, required) - status code that holds meaningful information about the result of inbound message processing.
-- `reasonPhrase` (string, optional) - status code that holds meaningful information about the result of inbound message processing.
-- `content` (byte[], optional) - content with configuration data.
+- `correlationId` (string, required): refer to [3/Messaging IPC][3/MIPC] RFC for description.<!--TODO-->
```

Comment:
It means that there should be a link to the particular section of that RFC, but since it's not proofread yet, I'll have to come back and amend the link. Or maybe not - then I'll remove the TODO commentary.